Source/ending.py

```diff
@@ -0,0 +1,96 @@
+# Author: Siena Okuno
+# Project: Interactive Programming
+# Date: 10.30.2017
+# Description: Determines endings
+
+import pygame
+import sys
+
```

Comment:
I wonder if these functions could be put into the GamePlay class. They seem to be important parts of it. Rather than importing into the file, adding them to the class makes more sense